### ISSUE TITLE
why these are zero ops ?  relu/dropout/pool

### ISSUE THREAD

**Issue by lk137095576** [Lyken17/pytorch-OpCounter#185]
`zero_ops = (
    nn.ReLU,
    nn.ReLU6,
    nn.Dropout,
    nn.MaxPool2d,
    nn.AvgPool2d,
    nn.AdaptiveAvgPool2d,
)`
zero op? there are no computation? or some reason?
